tracer/test/Datadog.Trace.Tests/LambdaCommonTests.cs

```diff
@@ -25,36 +27,44 @@ public class LambdaCommonTests
         public async Task TestCreatePlaceholderScopeSuccessWithTraceIdOnly()
         {
             await using var tracer = TracerHelper.CreateWithFakeAgent();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, "1234", null);
 
+            var myHeaders = new Dictionary<string, string> { { HttpHeaderNames.TraceId, "1234" }, };
+            var scope = LambdaCommon.CreatePlaceholderScope(tracer, myHeaders);
             scope.Should().NotBeNull();
             scope.Span.TraceId128.Should().Be((TraceId)1234);
             ((ISpan)scope.Span).TraceId.Should().Be(1234);
             scope.Span.SpanId.Should().BeGreaterThan(0);
         }
 
         [Fact]
-        public async Task TestCreatePlaceholderScopeSuccessWithSamplingPriorityOnly()
+        public async Task TestCreatePlaceholderScopeSuccessWith64BitTraceIdContext()
         {
             await using var tracer = TracerHelper.CreateWithFakeAgent();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, null, "-1");
+            var myHeaders = new Dictionary<string, string>
+            {
+                { HttpHeaderNames.TraceId, "1234" }, { HttpHeaderNames.SamplingPriority, "-1" }
+            };
+            var scope = LambdaCommon.CreatePlaceholderScope(tracer, myHeaders);
 
             scope.Should().NotBeNull();
-            scope.Span.TraceId128.Should().BeGreaterThan(TraceId.Zero);
-            ((ISpan)scope.Span).TraceId.Should().BeGreaterThan(0);
+            scope.Span.TraceId128.Should().Be((TraceId)1234);
+            ((ISpan)scope.Span).TraceId.Should().Be(1234);
             scope.Span.SpanId.Should().BeGreaterThan(0);
             scope.Span.Context.TraceContext.SamplingPriority.Should().Be(-1);
         }
 
         [Fact]
-        public async Task TestCreatePlaceholderScopeSuccessWithFullContext()
+        public async Task TestCreatePlaceholderScopeSuccessWith128BitTraceIdContext()
         {
             await using var tracer = TracerHelper.CreateWithFakeAgent();
-            var scope = LambdaCommon.CreatePlaceholderScope(tracer, "1234", "-1");
+            var myHeaders = new Dictionary<string, string>
+            {
+                { HttpHeaderNames.TraceId, "5744042798732701615" }, { HttpHeaderNames.SamplingPriority, "-1" }, { HttpHeaderNames.PropagatedTags, "_dd.p.tid=1914fe7789eb32be" }
+            };
+            var scope = LambdaCommon.CreatePlaceholderScope(tracer, myHeaders);
 
             scope.Should().NotBeNull();
-            scope.Span.TraceId128.Should().Be((TraceId)1234);
-            ((ISpan)scope.Span).TraceId.Should().Be(1234);
+            scope.Span.TraceId128.ToString().Should().Be("1914fe7789eb32be4fb6f07e011a6faf");
```

Comment:
This value agrees with dd-trace-java

Reply:
Confirmed that `1914fe7789eb32be4fb6f07e011a6faf` has
- `1914fe7789eb32be` in the upper 64 bits
- `4fb6f07e011a6faf` (`5744042798732701615`) in the lower 64 bits